filewriter_v2/README.md

```diff
@@ -0,0 +1,388 @@
+
+```
+/
+|
++-- entry
+    |
+    +-- data
+    +-- instrument
+    |   |
+    |   +-- beam
+    |   +-- detector
+    |       |
+    |       +-- threshold_1_channel*
+    |       +-- threshold_2_channel*
+    |       +-- difference_channel*
+    |       +-- geometry
+    |       +-- module
+    |       +-- transformations
+    |
+    +-- sample
+    |   |
+    |   +-- transformations
+    |   +-- beam
+    |
+    +-- source
+```
+\* this is not part of the NeXus standard yet, see [Multi-Channel Data](#multi-channel-data).
+
+## FileWriter V2 Configuration
+
+FileWriter V2 can be enabled and configured using the SIMPLON API.
+
+### SIMPLON FileWriter Configuration Parameters
+
+The configuration interface is accessible at `http://<ADDRESS_OF_DCU>/filewriter/api/1.8.0/config`.
+
+| Parameter | JSON Type | Default | Description |
+|-----------|------|---------|-------------|
+| `mode` | string | `disabled` | Enabled state of the FileWriter, either `enabled` or `disabled`. |
+| `compression_enabled` | bool | `true` | Optionally disable compression of image data. |
+| `image_nr_start` | uint | `1` | First image number on the axis `/entry/image_number`. |
+| `name_pattern` | string | `series_$id` | Basename of the files. `$id` will be replaced by the series number, i.e. using the default, the generated files will have the names:<br> `series_<series_number>_master.h5`<br> `series_<series_number>_data_<file_number>.h5`|
+| `nimages_per_file` | uint | `1000` | Maximum number of images stored in each data file. If set to `0`, all images are stored directly in the master file and no data files are created. If set to a value greater than zero, the images are stored in multiple data files. |
+| `v2_preview_enabled` | bool | `false` | Enabled state of the FileWriter V2 preview. |
+
+<!-- SIMPLON V2 is currently not ready for filewriter
+
+#### SIMPLON V2 FileWriter Configuration Parameters
+
+The configuration interface is accessible at `http://<ADDRESS_OF_DCU>/api/v2/daq/config`.
+
+| Parameter | JSON Type | Default | Description |
+|-----------|------|---------|-------------|
+| `enabled` | boolean | `false` | Enabled state of the FileWriter. |
+| `image_id_offset` | uint | `1` | First image number on the axis `/entry/image_number`. |
+| `name_pattern` | string | `series_$id` | Basename of the files. `$id` will be replaced by the series number, i.e. using the default, the generated files will have the names:<br> `series_<series_number>_master.h5`<br> `series_<series_number>_data_<file_number>.h5`|
+| `images_per_file` | uint | `1000` | Maximum number of images stored in each data file. If set to `0`, all images are stored directly in the master file and no data files are created. <br> If set to a value greater than zero, the images are stored in multiple data files, but are still accessible in the masterfile via the dataset "/entry/data/data", since this is a [Virtual Dataset](https://portal.hdfgroup.org/display/HDF5/Virtual+Dataset++-+VDS) in this case.|
```

Comment:
```suggestion
| `images_per_file` | uint | `1000` | Maximum number of images stored in each data file. If set to `0`, all images are stored directly in the master file and no data files are created. <br> If set to a value greater than zero, the images are stored in multiple data files, but are still accessible in the master file via the dataset "/entry/data/data", since this is a [Virtual Dataset](https://portal.hdfgroup.org/display/HDF5/Virtual+Dataset++-+VDS) in this case.|
```